CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ FEATURES:
 * Kubernetes 1.24 Support
   * Add support for Kubernetes 1.24 where ServiceAccounts no longer have long-term JWT tokens. [[GH-1431](https://github.com/hashicorp/consul-k8s/pull/1431)]
   * Upgrade kubeVersion in helm chart to support Kubernetes 1.21+.
+* Cluster Peering:
+  * Add support for setting failover `Targets` on the Service Resolver CRD.  [[GH-1284](https://github.com/hashicorp/consul-k8s/pull/1284)]
+  * Add support for redirecting to cluster peers on the Service Resolver CRD.  [[GH-1284](https://github.com/hashicorp/consul-k8s/pull/1284)]
```

Comment:
Does this change require a certain consul server version? If so, then we unfortunately need to hold off on merging for now because we're trying to keep the main branch in a releasable state.

Reply:
Yeah, it does. Thanks for the call out. I'll reach out for a review closer to the release.

Reply:
I think it's OK to review now, just calling out that we shouldn't merge until we're supporting 1.14.